src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -738,6 +751,7 @@ def ac_user_enable(_, username):
     try:
         user = mgr.ACCESS_CTRL_DB.get_user(username)
         user.enabled = True
+        mgr.ACCESS_CTRL_DB.save_attempt(username, 0)
```

Comment:
I'm wondering if it would make sense to reset the amount of attempts if the user changes the password?

Reply:
Yup, that makes sense. I have updated the PR. Now if user resets the password, attempts also gets reset.